.github/workflows/benchmark.yml

```diff
@@ -0,0 +1,38 @@
+name: Benchmarking
+
+on:
+  push:
+    branches:
+      - master
+    tags:
+      - "v*" 
+
+jobs:
+  benchmark-tag:
+    runs-on: ubuntu-latest
+    if: ${{ GITHUB_REF != 'master' }}
```

Comment:
I think this won't work - you need to use the `env` context, i.e. `env.GITHUB_REF`

Reply:
You can use `github.ref` also